tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventParseMethodContract.scala

```diff
@@ -154,21 +182,76 @@ trait LinagoraCalendarEventParseMethodContract {
            |                "description": "description 123",
            |                "start": "2017-01-11T16:00:00",
            |                "end": "2017-01-11T17:30:00",
-           |                "utcStart":"2017-01-11T09:00:00Z",
-           |                "utcEnd":"2017-01-11T10:30:00Z",
+           |                "utcStart": "2017-01-11T09:00:00Z",
+           |                "utcEnd": "2017-01-11T10:30:00Z",
            |                "timeZone": "Asia/Ho_Chi_Minh",
            |                "duration": "PT1H30M",
-           |                "location": "Hangout"
+           |                "location": "Hangout",
+           |                "organizer": {
+           |                    "name": "Raphael OUAZANA",
+           |                    "mailto": "ouazana@domain.tld"
+           |                },
+           |                "participants": [
+           |                    {
+           |                        "name": "Matthieu EXT_BAECHLER",
+           |                        "mailto": "baechler@domain.tld",
+           |                        "kind": "INDIVIDUAL",
+           |                        "role": "ADMIN",
+           |                        "participationStatus": "NEEDS-ACTION",
```

Comment:
Should we lowercase these fields?

Reply:
The spec is not clear should be lower or upper, so we should keep it original